packages/ui/src/core/input/input.component.tsx

```diff
@@ -41,6 +41,7 @@ const Input = intrinsicComponent<InputProps, HTMLInputElement>(
       iconStart,
       iconEnd,
       iconChange,
+      extraContent,
```

Comment:
Instead of naming extraContent, better to name it inputActions to make it clear this props used for this